packages/extension-chakra-storefront/jest-setup.js

```diff
@@ -144,6 +144,13 @@ jest.mock('@salesforce/cc-datacloud-typescript', () => ({
 global.TextEncoder = require('util').TextEncoder
 global.TextDecoder = require('util').TextDecoder
 
+// Add ResizeObserver polyfill for Chakra UI testing
```

Comment:
Chakra v3 uses modern browser API called ResizedObserver. Need to mock it for jest
![image](https://github.com/user-attachments/assets/ebbe5ff6-5a79-4050-9823-9158e6357567)